example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
+
+interface UseGravityArgs {
+  moverMass: number;
+  attractorMass: number;
+  attractor: Vector<number>;
+}
+
+const applyForceForStep = (
+  config: UseGravityArgs,
+  mover: Entity,
+  entity: Entity
+) => {
+  const force = forceV({
+    ...config,
+    mover: mover.position
+  });
+
+  return applyForce({
+    force,
+    moverMass: config.moverMass,
+    acceleration: mover.acceleration,
+    velocity: mover.velocity,
+    position: mover.position
+  });
+};
+
+const run = (config: UseGravityArgs) => {
+  const { start, stop } = rAF();
+
+  start((timestamp, lastFrame, mover) => {
+    const diffTime = timestamp > lastFrame + 64 ? 0 : lastFrame;
```

Comment:
This is a neat little trick from `react-spring`. Basically, if we dropped four or more frames, we just fast forward to applying the latest force.

Reply:
yea this is dope